test/configuration_policy_prune.go

```diff
@@ -6,9 +6,9 @@ package test
 import (
 	"errors"
 
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
-	. "github.com/stolostron/governance-policy-framework/test/common"
+	"github.com/onsi/ginkgo/v2"
```

Comment:
We should be allowing exceptions for dot imports of ginkgo and gomega.

I think the issue is that this file doesn't end with `_test.go`, so the ignore doesn't happen. Feel free to change the ignore to be global in this repo since it's a testing repo.